apps/next/app/server/util/functions.ts

```diff
@@ -0,0 +1,3 @@
+export const getDirectoryNameForUps = () => {
+  return "ups";
+}
```

Comment:
this can exist in the same file as the api route... also why is it a function?

Reply:
- Yes it can probably just be a named constant. Will change to that.
- Kept it outside the the API route because I expect other routes to need this info, so they can import it from this (impartial/general?) file instead of from the said route.